Add link to informal ideas (#157)
GEM-benchmark/NL-Augmenter: README.md

* [How do I create a filter?](#how-do-i-create-a-filter)
* [Motivation](docs/doc.md#motivation)
* [Review Criteria for Accepting Submissions](docs/doc.md#review-criteria-for-submissions)

> * [Informal And Untested Suggestions for Transformations](#informal-and-untested-suggestions-for-transformations)

Reviewer: I would order the table of contents the same way the text is ordered. So put this right before the Installation bullet.

I would also simplify the section title, to something like:
```suggestion
* [Some Ideas for Transformations](#some-ideas-for-transformations)
```

Reviewer 2: I agree the overall change is helpful. Also agree with Jascha's suggestion above!

## Colab notebook

<a href="https://colab.research.google.com/github/GEM-benchmark/NL-Augmenter/blob/main/notebooks/Write_a_sample_transformation.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a> To quickly see transformations and filters in action, run through our [colab notebook](https://colab.research.google.com/github/GEM-benchmark/NL-Augmenter/blob/main/notebooks/Write_a_sample_transformation.ipynb).

> ## Informal And Untested Suggestions for Transformations

Reviewer: ```suggestion
## Some Ideas for Transformations
```

> There are some ideas and previous papers discussed informally [here](https://github.com/GEM-benchmark/NL-Augmenter/issues/75) which maybe useful to look at. Note that up untill now there have already been a lot of implementations spanning across morphological inflections, character level changes & random noise. 

Reviewer: ```suggestion
If you need inspiration for what transformations to implement, check out https://github.com/GEM-benchmark/NL-Augmenter/issues/75, where some ideas and previous papers are discussed. So far, contributions have focused on morphological inflections, character level changes, and random noise. The best new pull requests will be dissimilar from these existing contributions.
```

git checkout -b my_awesome_transformation
```
We will base our transformation on an existing example.

> Create a new transformation directory by copying over an existing transformation. You can choose to copy from other [transformation directories](interfaces) depending on the task you wish to create a transformation for. Check some of the existing [pull requests](https://github.com/GEM-benchmark/NL-Augmenter/pulls?q=is%3Aopen+is%3Apr+label%3Atransformation) [and merged transformations](transformations) first to avoid duplicating efforts or creating transformations similar to previous ones.

Reviewer: ```suggestion
Create a new transformation directory by copying over an existing transformation. You can choose to copy from other [transformation directories](interfaces) depending on the task you wish to create a transformation for. Check some of the existing [pull requests](https://github.com/GEM-benchmark/NL-Augmenter/pulls?q=is%3Aopen+is%3Apr+label%3Atransformation) [and merged transformations](transformations) first to avoid duplicating efforts or creating transformations too similar to previous ones.
```

> Create a new transformation directory by copying over an existing transformation. You can choose to copy from other [transformation directories](interfaces) depending on the task you wish to create a transformation for. Check some of the existing [pull requests](https://github.com/GEM-benchmark/NL-Augmenter/pulls?q=is%3Aopen+is%3Apr+label%3Atransformation) [and merged transformations](transformations) first to avoid duplicating efforts or creating transformations similar to previous ones.

Reviewer 2: "Check out and browse through"?
